specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -3542,7 +3542,7 @@
           "VirtualMachines"
         ],
         "operationId": "VirtualMachines_SimulateEviction",
-        "description": "The operation to simulate the eviction of spot virtual machine. The eviction will occur within 30 minutes of calling the API",
+        "description": "The operation to simulate the eviction of spot virtual machine.",
```

Comment:
will the eviction time change? If yes, it's worth to call out so that user understand service better  since this is changing existing stable version.

Reply:
Now, Eviction will happen almost immediately after the API returns HTTP 204 Sucess. Also, The eviction happens with in-vm notification. Customer can subscribe to Azure metadata service to receive this notification